decidim-core/db/migrate/20191204075509_add_session_token_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddSessionTokenToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :decidim_users, :session_token, :string
```

Comment:
How and when is this column filled? Automatically by `devise`?

Reply:
It's used by `authenticatable_salt`, but it's nil at the beginning until the user first logs out. Then it's set, and every time they log out a new one is set, so that any sessions with the previous salt become invalid.

As seen in:
https://github.com/plataformatec/devise/issues/3031#issuecomment-403719013